### DISCUSSION TITLE
Server/middleware. No messages in the console.

### DISCUSSION THREAD

**Discussion by timoshechkin** [nuxt/nuxt#16446] [Questions]
Created a log.ts file in the folder (server/middleware) with the content: export default defineEventHandler((event) => {
   console.log('New request: ' + event.req.url)
}). But there are no messages in the console.

**Comment by TheAlexLichter** [comment]
Works fine [in my stackblitz](https://stackblitz.com/edit/nuxtframeworkdiscussions8777?file=server%2Fmiddleware%2Fmain.ts). Be aware that logs are shown in the server console only.

PS: Feel free to attack a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction) so people can help you easier and quicker 😋
